src/framework/module/wishlist.module.ts

```diff
@@ -0,0 +1,27 @@
+import { MikroOrmModule } from '@mikro-orm/nestjs';
+import { Module } from '@nestjs/common';
+import { WishlistDbEntity } from 'src/adapter/db/wishlist.entity';
+import { WishlistGateway } from 'src/adapter/db/wishlist.gateway';
+import { WishlistController } from 'src/adapter/web/wishlist.controller';
+import { WishlistService } from 'src/application/service/Wishlist.service';
+
+import { AuthGuardModule } from './auth.guard.module';
+import { ProductModule } from './product.module';
+
+@Module({
+  imports: [MikroOrmModule.forFeature([WishlistDbEntity]), AuthGuardModule, ProductModule],
+  controllers: [WishlistController],
+  providers: [
+    WishlistService,
+    WishlistGateway,
+    {
+      provide: 'WishlistUseCase',
+      useClass: WishlistService,
+    },
+    {
+      provide: 'WishlistGateway',
+      useExisting: WishlistGateway,
+    },
+  ],
```

Comment:
모듈간 의존성은 갖지 않도록 작성 부탁드릴게요!


```suggestion
  imports: [MikroOrmModule.forFeature([WishlistDbEntity, ProductDbEntity]), AuthGuardModule],
  controllers: [WishlistController],
  providers: [
    WishlistService,
    {
      provide: 'WishlistUseCase',
      useClass: WishlistService,
    },
    {
      provide: 'WishlistGateway',
      useExisting: WishlistGateway,
    },
    {
      provide: 'ProductGateway',
      useExisting: ProductGateway,
    }
  ],
```

Reply:
넵 수정하였습니다